src/utils.ts

```diff
@@ -0,0 +1,28 @@
+import {Point, UnorderdLineSegment} from './line-tools';
+
+/**
+ * This module provides utility functions for working with points and line segments.
+ * @module utils
+ */
+
+/**
+ * Creates unordered line segments from an array of points.
+ * The function connects each point to the next one in the array, and the last point back to the first one.
+ * This can be used to create a closed shape from a list of points.
+ *
+ * @function createLinesFromPoints
+ * @param {Point[]} points - An array of points.
+ * @returns {UnorderdLineSegment[]} - An array of unordered line segments created from the points.
+ */
+export function createLinesFromPoints(points: Point[]): UnorderdLineSegment[] {
```

Comment:
Just wondering if this is actually needed. Dont we just need the Points in the specific order?
like this:
` [p1, p2, p2, p3, p3, p4, ...]` would make it easier to read out just a Point list.
Not really sure.